src/serializationHelpers.ts

```diff
@@ -173,3 +328,523 @@ export function serializeYearMonth(yearMonth: string): Buffer {
     const serializedMonth = encodeWord8(month);
     return Buffer.concat([serializedYear, serializedMonth]);
 }
+
+/**
+ * @param bigint 128 bit bigint value that need to be splitted
+ * @returns two 64 bit bigint values left and right
+ * where left is first part of given 128 bigint value
+ * where right is second part of given 128 bigint value
```

Comment:
```suggestion
 * @param bigint 128 bit bigint value that needs to be split
 * @returns two 64 bit bigint values left and right
 * where left is first part of given 128 bigint value
 * where right is second part of given 128 bigint value
```

Reply:
I don't know if what first and second part means?
I would have expected it to explain which has the most significant bits, and which has the least significant bits.

Also I think the encoding of 128 bit might be incorrect for big endian, seeing as you serialize left and right in the same order in both cases?

Reply:
@shjortConcordium we have fixed encoding for 128 which needed LE values and we modified the function accordingly.